dbm/include/config.h

```diff
@@ -2,7 +2,7 @@
 /* include/config.h.in.  Generated from configure.ac by autoheader.  */
 
 /* A compiler identification string */
-#define COMPILER_STRING "Compiled with g++-9 -Wall -O2 -DNDEBUG   -DBOOST_DISABLE_THREADS"
+#define COMPILER_STRING "Compiled with g++-10    -DBOOST_DISABLE_THREADS"
```

Comment:
are we sure we dont want to compile with optimization (-O2) here?

Reply:
that file is auto generated by autotools. I think it's best to leave it as-is

Reply:
I think we should leave it for now. In my mind we're going to be porting it into rust before we start worrying about performance anyway.